website/settings/defaults.py

```diff
@@ -31,8 +31,13 @@ def parent_dir(path):
 with open(os.path.join(APP_PATH, 'package.json'), 'r') as fobj:
     VERSION = json.load(fobj)['version']
 
-# Hours before email confirmation tokens expire
-EMAIL_TOKEN_EXPIRATION = 24
+# Expiration time for verification key
+EXPIRATION_TIME_DICT = {
+    'password': 30,         # 30 minutes for forgot and reset password
+    'confirm': 24 * 60,     # 24 hours in minutes for confirm account and email
+    'claim': 30 * 24 * 60   # 30 days in minutes for claim contributor-ship
```

Comment:
set 30 days instead of 7 days claim token, change back to 7 days after resend claim confirmation is implemented: [OSF-6998](https://openscience.atlassian.net/browse/OSF-6998).